app/views/nsm/steps/case_outcome/edit.html.erb

```diff
@@ -0,0 +1,40 @@
+<% title t(".page_title") %>
+<% step_header(path: nsm_applications_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_error_summary(@form_object) %>
+
+    <%= step_form @form_object do |f| %>
+      <%= f.govuk_radio_buttons_fieldset :case_outcome, legend: { size: 'xl', class: 'govuk-!-margin-bottom-6' } do %>
```

Comment:
So this technically isn't to spec based on [GDS guidelines on how to use the radio buttons](https://design-system.service.gov.uk/components/radios/). 
<img width="562" alt="image" src="https://github.com/user-attachments/assets/0ba15f0d-259c-43ad-b480-7bfb60fbc120">
You can use my PR as an example: 
<img width="634" alt="image" src="https://github.com/user-attachments/assets/be4a1d50-2a5d-42cd-9284-54867ed7eb30">